README.md

```diff
@@ -16,3 +16,10 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+
+To test the sorting algorithm, I would use reverse-sorted lists (for worst case runtimes as the most amount of movement/comparisons would need to be done as every "chunk" is "unsorted") and run linearly larger lists through the algorithm and compare their runtimes. For example, I would run a reverse-sorted array of length 100, then of 200, then 300, etc. I would compare the runtimes to see if they increase linearly. If the runtime differences are negligaile, I can work with factors of thousands or ten-thousands for the input sizes. I would similarly compare their run times for being linear. If an input of size n has a runtime of around t, I would expect an input size of 10n to have a runtime of around 10t. I may start out with larger input sizes to reach the asymptotic complexity of the runtime of the sorting algorithm. 
+
+As there are an arbitrary amount of elements in the list, a radix sort may have been implemented. This has a theoretical potential of linear runtime complexity, as each item would only need to be read once as it would be sorted into its corresponding bucket. However, because the researcher's sort is said to be comparison based between two items, it would not be able to be this type of sorting. Thus I do not think that X could be correct.
```

Comment:
And what do you know about the complexity of the general sorting problem that's relevant here?